edgelet/iotedge/src/check/checks/container_connect_upstream.rs

```diff
@@ -139,16 +139,18 @@ impl ContainerConnectUpstream {
             &port,
         ]);
 
-        let proxy = settings
-            .agent()
-            .env()
-            .get("https_proxy")
-            .map(std::string::String::as_str);
-        self.proxy = proxy.map(ToOwned::to_owned);
-        if let Some(proxy) = proxy {
-            args.extend(&["--proxy", proxy]);
+        if port == 443.to_string() {
```

Comment:
You could also do 
```suggestion
        if &port == "443" {
```

Reply:
If a customer decides to use a port other than 443 for https this check wouldn't run, but I think that's probably fine, I doubt many customers are doing that.

Reply:
Yes, I though about that but I am not sure you can change that port in edgeAgent.